### PR TITLE
Dm/data fixes

### PR DESCRIPTION
**Pull request by danielmend**
fix spikes in training + make sure stuff is resampled/shuffled properly

### CODE REVIEW

File: src/training/data.py [hunk 1]
```diff
@@ -357,17 +357,25 @@ def get_data(args, preprocess_fns, epoch=0):
 
     if args.train_data:
         data["train"] = get_wds_dataset(
-            args, preprocess_train, is_train=True, epoch=epoch)
+            args, preprocess_train, is_train=True, epoch=epoch, floor=False)
     if args.val_data:
         data["val"] = get_wds_dataset(
-            args, preprocess_val, is_train=False)
+            args, preprocess_val, is_train=False, floor=False)
```
Comment (iejMac) [inline src/training/data.py:363]: why do we need this?

File: src/training/data.py [hunk 2]
```diff
@@ -357,17 +357,25 @@ def get_data(args, preprocess_fns, epoch=0):
...
     if args.image_data:
-        BUFFER = 0
+        np.random.seed(args.seed + epoch) # reseed every epoch
+        start_loc = np.random.randint(0, 1e7)
```
Comment (iejMac) [inline src/training/data.py:366]: why 1e7, this seems very arbitrary

File: src/training/loss.py [hunk 3]
```diff
@@ -110,7 +110,7 @@ def forward(self, image_features, text_features, logit_scale):
                 self.prev_num_logits = num_logits
         else:
             labels = self.labels[device]
-
+        #print(logits_per_image.shape)
```
Comment (iejMac) [inline src/training/loss.py:113]: cleanup

File: src/training/train.py [hunk 4]
```diff
@@ -57,8 +58,17 @@ def train_one_epoch(model_video, data, epoch, optimizer, scaler, scheduler, args
         scheduler(step)
 
         embeddings, toks, meta = batch
+        if i < 5:
+            print(meta[0])
+        #print(embeddings.shape)
         embeddings = embeddings.to(device, non_blocking=True)
         toks = toks.to(device, non_blocking=True)
+        #print(toks)
+        #print(meta[0])
```
Comment (iejMac) [inline src/training/train.py:67]: cleanup

File: src/training/data.py (hunk 2 above)
Comment (danielmend) [reply src/training/data.py:366]: yea it is arbitrary. what do u suggest i replace 1e7 with? im thinking maybe an arg or something

File: src/training/main.py [hunk 5]
```diff
@@ -124,8 +124,8 @@ def main():
     elif args.report_to == "wandb" and is_master(args):
         logging.debug("Starting wandb.")
         wandb.init(
-            project="",
-            entity="", #TODO: do you need this?
+            project="laion-video-clip",
+            entity="danielmend", #TODO: do you need this?
```
Comment (danielmend) [inline src/training/main.py:128]: i will remove this right before merging

File: src/training/data.py (hunk 2 above)
Comment (iejMac) [reply src/training/data.py:366]: something like args.img_train_num_samples - samples_per_worker
